### ISSUE TITLE
Add more descriptive info about the application

### ISSUE THREAD

**Issue by mcanoy** [rht-labs/api-design#14]
Here is the relevant output of the code gen for SpringBoot

```
        return new ApiInfoBuilder()
            .title("EMdP modeling and API design")
            .description("No descripton provided (generated by Swagger Codegen https://github.com/swagger-api/swagger-codegen)")
            .license("")
            .licenseUrl("")
            .termsOfServiceUrl("")
            .version("0.2.0")
            .contact(new Contact("","", ""))
            .build();
    }
```

**Comment by sherl0cks** [comment]
This code is the output of swagger codegen. We need to add the empty items in the spec to be complete.

**Comment by sherl0cks** [comment]
resolved by #26
